app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -3257,7 +3257,7 @@ rf-navbar-search {
 .menu-separator {
     height: .2em;
     width: 100%;
-    border-top: 1 px solid #ccc;
+    border-top: 1px solid #ccc;
 }
 
 rf-project-layers-page {
```

Comment:
You can just use these styles instead of the ones you added down below. I've been adding each of these list pages to this list for now.

```css
rf-project-layers-page,
rf-project-analyses-page {
...
}
```

Reply:
☝️